### ISSUE TITLE
No "empty" color

### ISSUE THREAD

**Issue by servocoder** [kartik-v/yii2-widget-colorinput#14]
I am unable to display "empty" color by default for new model.
It works if I use plugin directly like:
```
$(selector).spectrum({
    allowEmpty: true
});
```
But there is no effect when I set widget options as
```
'pluginOptions'=> [
    'allowEmpty' => true,
]
```
Check the link below for details:
https://bgrins.github.io/spectrum/#options-color

**Comment by kartik-v** [comment]
The `allowEmpty` only works if you have input type set to `text` OR something other `color`. This will need a change to the widget code and will check this later.

**Comment by kartik-v** [comment]
This will be resolved via #16
